woocommerce/wc-admin
# Try: Replace Accordion with Panel component

Attempts to replace the `Accordion` with the `@wordpress/components` `Panel` component.

I'm not sure behind the decisions to create the `Accordion` component so we'd need to double check before merging this.

Note that a test is failing related to the panel being open.  I think this may be due to how GB sets the open state and it's not ready by the time the tests render.

### Screenshots

**Before**
<img width="688" alt="Screen Shot 2020-12-25 at 3 28 29 PM" src="https://user-images.githubusercontent.com/10561050/103141692-de7a7100-46c5-11eb-99e5-f4b0d4281cd3.png">


**After**

<img width="704" alt="Screen Shot 2020-12-25 at 3 23 24 PM" src="https://user-images.githubusercontent.com/10561050/103141646-2c42a980-46c5-11eb-918f-21794b86a64d.png">


### Detailed test instructions:

1. Open the activity panel.
2. Check that styles match expected. (note some are more closely following GB `Panel` component.
3. Try with/without:
* Orders that need fulfilling
* Reviews
* Low stock products
